fe/fe-core/src/main/java/com/starrocks/planner/PaimonScanNode.java

```diff
@@ -132,7 +148,43 @@ public void setupScanRangeLocations(TupleDescriptor tupleDescriptor, ScalarOpera
         scanNodePredicates.setSelectedPartitionIds(selectedPartitions.values());
     }
 
-    private void addScanRangeLocations(DataSplit split, String predicateInfo) {
+    private THdfsFileFormat fromType(String type) {
+        THdfsFileFormat tHdfsFileFormat;
+        switch (type) {
```

Comment:
could you need to convert it to lower case?

Reply:
There's no need, as the Paimon SDK will provide the format in lowercase directly.